### PR TITLE
Add checks for deprecated modules

### PR DESCRIPTION
**Pull request by ekohl**

### CODE REVIEW

File: .github/workflows/test.yml.erb [hunk 1]
```diff
@@ -0,0 +1,29 @@
+name: Test
```
Comment (bastelfreak) [inline .github/workflows/test.yml.erb:1]: is that an .erb file on purpose?

Comment (ekohl) [reply .github/workflows/test.yml.erb:1]: No, certainly not.

**Comment by bastelfreak** [comment]
works for me:
```
bastelfreak@bastelfreak-nb ~/code/modulesync_config/modules/voxpupuli/puppet-borg $ bundle exec metadata-json-deps metadata.json -v
Checking metadata.json
  saz/ssh (>= 4.0.0 < 9.0.0) matches 8.0.0
  puppetlabs/vcsrepo (>= 2.0.0 < 6.0.0) matches 5.0.0
  puppet/systemd (>= 0.4.0 < 4.0.0) matches 3.5.2
  puppetlabs/stdlib (>= 4.25.1 < 9.0.0) matches 8.1.0
  puppet/ssh_keygen (>= 4.1.0 < 6.0.0) matches 5.0.1
bastelfreak@bastelfreak-nb ~/code/modulesync_config/modules/voxpupuli/puppet-borg $ vim metadata.json
bastelfreak@bastelfreak-nb ~/code/modulesync_config/modules/voxpupuli/puppet-borg $ bundle exec metadata-json-deps metadata.json -v
Checking metadata.json
  saz/ssh (>= 4.0.0 < 9.0.0) matches 8.0.0
  puppetlabs/vcsrepo (>= 2.0.0 < 6.0.0) matches 5.0.0
  camptocamp/systemd was superseded by puppet-systemd
  puppetlabs/stdlib (>= 4.25.1 < 9.0.0) matches 8.1.0
  puppet/ssh_keygen (>= 4.1.0 < 6.0.0) matches 5.0.1
bastelfreak@bastelfreak-nb ~/code/modulesync_config/modules/voxpupuli/puppet-borg $ git diff
diff --git a/Gemfile b/Gemfile
index b6dcf45..4dd64cb 100644
--- a/Gemfile
+++ b/Gemfile
@@ -30,5 +30,5 @@ gem 'facter', ENV['FACTER_GEM_VERSION'], :require => false, :groups => [:test]

 puppetversion = ENV['PUPPET_VERSION'] || '>= 6.0'
 gem 'puppet', puppetversion, :require => false, :groups => [:test]
-
+gem 'metadata_json_deps', git: 'https://github.com/ekohl/metadata_json_deps', branch: 'check-deprecated-modules'
 # vim: syntax=ruby
diff --git a/metadata.json b/metadata.json
index 5fb568a..619b8ff 100644
--- a/metadata.json
+++ b/metadata.json
@@ -24,7 +24,7 @@
       "version_requirement": ">= 2.0.0 < 6.0.0"
     },
     {
-      "name": "puppet/systemd",
+      "name": "camptocamp/systemd",
       "version_requirement": ">= 0.4.0 < 4.0.0"
     },
     {
bastelfreak@bastelfreak-nb ~/code/modulesync_config/modules/voxpupuli/puppet-borg $
```

I'm fine with merging it after the CI is fixed.

**Comment by ekohl** [comment]
Not entirely unsurprising, but now CI actually runs :D

**Pull request merged**